src/directives/infinite-scroll/infinite-scroll-load-button.directive.ts

```diff
@@ -1,4 +1,4 @@
-import { Directive, ElementRef, Input, Output, Renderer2, TemplateRef, ViewContainerRef } from '@angular/core';
+import { Directive, Input, Output, Renderer2, TemplateRef, ViewContainerRef } from '@angular/core';
 import { Observable, Subject } from 'rxjs';
 
 @Directive({
```

Comment:
Getting an expression changed error:

ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value for 'attr.aria-busy': 'false'. Current value: 'true'.

Might not be caused by this and could be a separate issue